packages/typespec-ts/test/azureIntegration/azureArmResources.spec.ts

```diff
@@ -454,4 +544,478 @@ describe("Azure Arm Resources Rest Client", () => {
     assert.strictEqual(result[0]?.name, validNestedResource.name);
     assert.strictEqual(result[0]?.type, validNestedResource.type);
   });
+
+  // location resource
+  it("should get LocationResources ", async () => {
+    const result = await client
+      .path(
+        "/subscriptions/{subscriptionId}/providers/Azure.ResourceManager.Resources/locations/{location}/locationResources/{locationResourceName}",
+        SUBSCRIPTION_ID_EXPECTED,
+        LOCATION_EXPECTED,
+        "resource"
+      )
+      .get();
+    assert.strictEqual(result.status, "200");
+    assert.deepEqual(result.body, validLocationResource);
```

Comment:
Could you help check if deepEqual would validate type information?

Reply:
Updated to `deepStrictEqual`